src/readline.cr

```diff
@@ -1,6 +1,8 @@
 @[Link("readline")]
 {% if flag?(:openbsd) %}
 @[Link("termcap")]
+{% elsif flag?(:darwin) %}
+@[Link("ncurses")]
```

Comment:
Why does this regard static compilation?

Reply:
If ncurses isn't linked with readline when statically compiling, you get missing symbol errors as it depends on it.